src/coreclr/jit/lowerxarch.cpp

```diff
@@ -3429,7 +3429,7 @@ void Lowering::LowerHWIntrinsicDot(GenTreeHWIntrinsic* node)
                     //   e6, e7, e4, e5 | e2, e3, e0, e1
                     //   e7, e6, e5, e4 | e3, e2, e1, e0
 
-                    shuffleConst = 0x4D;
+                    shuffleConst = 0x4E;
```

Comment:
Was this just wrong before?

Reply:
Yes, see https://github.com/dotnet/runtime/pull/64140#discussion_r790132513

Reply:
I was too fast then, should have checked the timestamps on the comments.